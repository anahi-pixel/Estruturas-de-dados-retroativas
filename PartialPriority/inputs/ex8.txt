1 80 100
1 40 200
1 60 300
1 20 400
1 70 50
1 50 150
1 30 250
1 10 700
1 90 350
7
5
6
2 15
7
5
6
2 25
7
5
6
2 35
7
5
6