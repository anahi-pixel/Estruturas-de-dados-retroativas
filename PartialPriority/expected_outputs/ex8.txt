1 Fila de prioridade no instante atual: 
50 100 150 200 250 300 350 400 700 
50
9
2 Fila de prioridade no instante atual: 
50 100 150 200 250 300 350 400 
50
8
3 Fila de prioridade no instante atual: 
50 100 150 200 250 300 350 
50
7
4 Fila de prioridade no instante atual: 
50 100 150 200 300 350 
50
6
